sw/SoC/examples/blinky/Makefile

```diff
@@ -17,6 +17,8 @@ LD_SCRIPT =       $(SOC_SW_ROOT_DIR)/linker/$(MCU)_FLASH.ld
 INC = 		      -Iinc
 STARTUP = 	      startup
 
+LIBLIST = 
```

Comment:
this var is empty, do we need it or is just spurious code?

Reply:
In Blinky it is not required, I will delete it. However, it is left empty in the template since it should be filled with static libraries in a project requiring them.